R/summariseClinicalRecords.R

```diff
@@ -44,19 +46,17 @@
 #'summarisedResult |> print()
 #'PatientProfiles::mockDisconnect(cdm = cdm)
 #'}
-summariseClinicalRecords <- function(omopTable,
-                               recordsPerPerson = c("mean", "sd", "median", "q25", "q75", "min", "max"),
-                               inObservation = TRUE,
-                               standardConcept = TRUE,
-                               sourceVocabulary = FALSE,
-                               domainId = TRUE,
-                               typeConcept = TRUE) {
-
+summariseClinicalRecords <- function(cdm,
+                                     tableNames,
```

Comment:
tableName, in principle arguments will always be singular

Reply:
In fact omopTableName? @KimLopezGuell and @martaalcalde  to decide

Reply:
I like "omopTableName"